cli-staking/src/main.rs

```diff
@@ -266,12 +266,12 @@ fn transfer_admin(
 
 fn initialize_vault(
     program: &Program,
-    base: &String,
     admin: &Keypair,
     token_mint: &Pubkey,
 ) -> Result<()> {
-    let base_keypair = read_keypair_file(base).expect("Base keypair file not found");
+    let base_keypair = Keypair::new(); // throwable keypair
```

Comment:
I personally prefer to pass the `path/to/base_keypair.json` from CLI instead of generating on the fly. The reason is because if user forgotten to record the `print`-ed base key, they can't derive the vault PDA anymore. They have to search at explorer. But, either way works for me

Reply:
I think the function is not expected to be called by normal user. For admin, who setup to pool, they should be aware that, so I will keep it as the same

Reply:
Alright, no problem